code/modules/hydroponics/grown/citrus.dm

```diff
@@ -33,6 +33,59 @@
 	filling_color = "#00FF00"
 	juice_results = list(/datum/reagent/consumable/limejuice = 0)
 
+//3D Lime
+/obj/item/seeds/lime_3d
+	name = "pack of extradimensional lime seeds"
+	desc = "A product of research into the overlapping of 3D dimensions."
+	icon_state = "seed-lime3d"
+	species = "lime"
+	plantname = "Extradimensional Lime Tree"
+	product = /obj/item/reagent_containers/food/snacks/grown/citrus/lime_3d
+	lifespan = 60
+	endurance = 50
+	yield = 5
+	potency = 20
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_grow = "lime-grow"
+	icon_dead = "lime-dead"
+	genes = list(/datum/plant_gene/trait/repeated_harvest)
+	reagents_add = list(/datum/reagent/consumable/nutriment = 0.1, /datum/reagent/consumable/nutriment/vitamin = 0.15, /datum/reagent/consumable/sodiumchloride = 0.15)
+	rarity = 30
+
+/obj/item/reagent_containers/food/snacks/grown/citrus/lime_3d
+	seed = /obj/item/seeds/lime_3d
+	name = "extradimensional lime"
+	desc = "You can hardly wrap your head around this thing."
+	icon_state = "lim"
+	filling_color = "#00FF00"
+	juice_results = list(/datum/reagent/consumable/limejuice = 0)
+	volume = 150
+	distill_reagent = /datum/reagent/consumable/ethanol/triple_sec
+	tastes = list("polygons" = 1, "limes" = 1)
+
+/obj/item/reagent_containers/food/snacks/grown/citrus/lime_3d/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
+	var/max = rand(10, 20)
+
+	for (var/counter = 1 to max)
+		var/obj/item/reagent_containers/food/snacks/grown/citrus/lime/lime = new(get_turf(src))
+		lime.throw_at(pick(oview(7, get_turf(src))), 10, 1)
+		lime.seed.lifespan = seed.lifespan
+		lime.seed.endurance = seed.endurance
+		lime.seed.maturation = seed.maturation
+		lime.seed.production = seed.production
+		lime.seed.yield = seed.yield
+		lime.seed.potency = seed.potency
+		lime.seed.weed_rate = seed.weed_rate
+		lime.seed.weed_chance = seed.weed_chance
+		lime.seed.genes |= seed.genes
+		for(var/datum/reagent/R  in reagents.reagent_list)
+			R.volume /= 2
```

Comment:
why is it only cutting the reagents in half, that would make it duplicate reagents

Reply:
i tried dividing to the total splitted slimes but the reagents would be real close to 0 which would result in none

Reply:
like 0.00000000000032e-7

Reply:
either way doesnt matter, because botanist can already duplicate chems with the 3d lime